packages/backend/src/accounting/tigerbeetle/service.ts

```diff
@@ -118,24 +120,26 @@ export async function createLiquidityAccount(
 
 export async function createSettlementAccount(
   deps: ServiceDependencies,
-  ledger: number
-): Promise<void> {
+  account: SettlementAccount
+): Promise<SettlementAccount> {
   try {
     await createAccounts(deps, [
       {
-        id: ledger,
-        ledger,
+        id: account.asset.ledger.toString(),
```

Comment:
We're now going number -> string -> bigint
How bad would it be to use `AccountId` in the base `AccountService? :grimacing: :grimacing: :grimacing:
https://github.com/interledger/rafiki/blob/d8800feba065bcd875db363d1d1472f146f08c4c/packages/backend/src/accounting/tigerbeetle/utils.ts#L3

Reply:
🤔 hMMM I might be complicating it. Maybe I can just store `ledger` on the `LedgerAccount` and then do an `Access` lookup to get the information I need without changing the function signature